finalProject/10_speckled_frogs.md

```diff
@@ -1,31 +1,18 @@
-# 10 Speckled Frogs
-
-Create a file named `tenSpeckledFrogs.js` and within that file, write several a program that will print the following nursery rhyme:
-
-> 3 speckled frogs sat on a log 
-> eating some most delicious bugs.  
-> One jumped in the pool where its nice and cool,
-> then there were 2 speckled frogs.
->
-> 2 speckled frogs sat on a log 
-> eating some most delicious bugs.  
-> One jumped in the pool where its nice and cool,  
-> then there was 1 speckled frog.
->
-> 1 speckled frog sat on a log 
-> eating some most delicious bugs.  
-> One jumped in the pool where its nice and cool,  
-> then there were no more speckled frogs!
-
-## Required
-
-Make your program print the rhyme above for *10* frogs, with attention to where language changes. Notice the very small differences between what should be printed out when 3, 2 or 1 frog is sitting on a log.
-
-## Extension 1
-
-Make your program work for _any_ number of frogs.
-
-## Extension 2
-
-Print word versions of each number in the first and fourth lines, for example, the first verse in the above example would print 'Three speckled frogs...' and 'were two speckled frogs'. There are solutions that don't involve using a library or a huge piece of someone else's code.
-
```

Comment:
Nice job on this one! Have you tried either of the extensions?

Reply:
@mschae16
Thank you! I have not yet, but plan on looking at them in the next week or so when I have some more time to work through it.